app/javascript/components/collections/AddToCollections.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div
+    class="AddRepoToCollections absolute top-0 right-[120px] md:relative md:right-0 md:pl-5 md:pb-4 z-20"
+  >
+    <div
+      v-if="settingsVisibility"
```

Comment:
update name from 'settingsVisibility' to 'canManage'

Reply:
done